### PR TITLE
Update New-CsTeamsMessagingPolicy.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
per discussion with PM John Taig

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ New-CsTeamsMessagingPolicy [-AllowOwnerDeleteMessage <Boolean>] [-AllowSmartRepl
 ## DESCRIPTION
  The CsTeamsMessagingPolicy cmdlets enable administrators to control if a user is enabled to exchange messages. These also help determine the type of messages users can create and modify.  This cmdlet creates a new Teams messaging policy.  Custom policies can then be assigned to users using the Grant-CsTeamsMessagingPolicy cmdlet.
 
+
+**NOTE** If the value from allowuserdeletemessage is set to false, the team owner will not be able to delete his own messages
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md:34]:
```suggestion
**Note**: If the -allowuserdeletemessage parameter is set to FALSE, the team owner will not be able to delete his own messages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RuiPereiraTabares, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by RuiPereiraTabares** [comment]
@get-itips even if allowownerdeletemessage is on true the teams owner will not be able to delete the messages his sent in a team channel if the allowuserdeletemessage is on false

**Comment by get-itips** [comment]
@RuiPereiraTabares Rui, I can not edit your PR.
@dariomws @tseward Where do you think this note should belong? Thanks

**Comment by tseward** [comment]
I think this should go under the parameter description.

File: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ New-CsTeamsMessagingPolicy [-AllowOwnerDeleteMessage <Boolean>] [-AllowSmartRepl
 ## DESCRIPTION
  The CsTeamsMessagingPolicy cmdlets enable administrators to control if a user is enabled to exchange messages. These also help determine the type of messages users can create and modify.  This cmdlet creates a new Teams messaging policy.  Custom policies can then be assigned to users using the Grant-CsTeamsMessagingPolicy cmdlet.
 
+**Note**: If the value from allowuserdeletemessage is set to FALSE, the team owner will not be able to delete his own messages.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md:33]:
```suggestion
**Note**: If the `-AllowOwnerDeleteMessage` parameter is set to FALSE, the team owner will not be able to delete his own messages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@RuiPereiraTabares - Apologies, final suggested change, please. Also, could you please move the note to below the parameter description as suggested by @tseward? Many thanks.

**Comment by RuiPereiraTabares** [comment]
> @RuiPereiraTabares - Apologies, final suggested change, please. Also, could you please move the note to below the parameter's description as suggested by @get-itips and @tseward? Many thanks.
@JohanFreelancer9 
Done

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Thank you @RuiPereiraTabares. However, please insert the blank lines so that the preceding text does not flow into the note. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
Approve but I think this sentence should be in the AllowOwnerDeleteMessage description or at least both.

File: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JohanFreelancer9) [reply skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md]: Also, please change "his" to "their" to keep it gender-neutral.

**Comment by yogkumgit** [comment]
@RuiPereiraTabares

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Pull request merged**